### DISCUSSION TITLE
What happened to RTL guidelines?

### DISCUSSION THREAD

**Discussion by dwelle** [mdn/mdn-community#595] [Content] [answered]
There's a developer guidelines on RTL we've been using over at Excalidraw, but the link is dead: https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/RTL_Guidelines. A snapshot can be accessed at https://web.archive.org/web/20210428214405/https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/RTL_Guidelines

It's strange because I can't find it ever even was on https://github.com/mdn/content/blob/main/files/en-us/web/guide/index.md.

What happened? Thanks!

**Comment by Rumyra** [comment, marked as answer]
Hi @dwelle - we moved a bunch of docs from MDN a while ago - these can be found here https://firefox-source-docs.mozilla.org/ 

There's also a capture of some of them in our archive repository here https://github.com/mdn/archived-content/tree/main/files/en-us/mozilla/developer_guide

I hope that helps!

**Comment by pepelsbey** [comment]
Closing as answered/settled.
